xml/System.Text.Json/JsonSerializerOptions.xml

```diff
@@ -72,9 +81,17 @@
         <ReturnType>System.Collections.Generic.IList&lt;System.Text.Json.Serialization.JsonConverter&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets the list of user-defined converters that were registered.</summary>
+        <value>The list of custom converters.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Once serialization or deserialization occurs, the list cannot be modified.
```

Comment:
It should state, "should not be modified", not cannot. There is nothing preventing the caller from modifying the list. The changes just won't be reflected when (de)serializing.

cc @steveharter

Reply:
The caller cannot modify the list. They will get an InvalidOperationException.

Reply:
Really? We should document the exception then?